### ISSUE TITLE
Received ERC20 assets activities are not showing up

### ISSUE THREAD

**Issue by jagodarybacka** [tallycash/extension#2082]
New activities of incoming ERC20 tokens transfers are not showing up in the activities list in the expected time.

### Steps to reproduce

- send ERC20 token from one address to another
- send activity is showing up instantly
- receive activity on the other account is not showing up for a long time

**Comment by mhluongo** [comment]
Against which version @jagodarybacka?

**Comment by mr-michael** [comment]
@jagodarybacka are you still seeing this in 1.61? How long is it taking?

**Comment by jagodarybacka** [comment]
@mr-michael yes it is still a case, I checked today and "received" activity is not added to the list at all even after >2h, tbh no "received" activities are on the list at all so they are probably not fetched correctly 🤔
